ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.12-nightly.yaml

```diff
@@ -85,6 +85,16 @@ tests:
     test:
     - ref: openshift-qe-ingress-perf
     workflow: rosa-aws-sts
+- as: perfscale-rosa-multiaz-network-perf-3nodes-periodic
+  cron: 0 13 * * 5
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      CHANNEL_GROUP: nightly
+      OPENSHIFT_VERSION: "4.12"
```

Comment:
Can we add other applicable envs here? ENABLE_SCALING is ON by default. Is that what you want to test with? There might be other similar env variables to watch out for.  Explicitly setting some env variables might avoid confusion.

Also, if we want to test with default nodes, can we update test name to defaultnodes instead of 3nodes.

Reply:
Ack, will update them.

Reply:
Updated, ptal.